package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jquery-focuspoint",
+  "version": "1.1.3",
+  "description": "jQuery plugin for 'responsive cropping'. Dynamically crop images to fill available space without cutting out the image's subject. Great for full-screen images.",
+  "main": "js/jquery.focuspoint.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jonom/jquery-focuspoint.git"
+  },
+  "keywords": [
+    "\"crop\"",
+    "\"cropping\"",
+    "\"image\"",
+    "\"responsive\"",
+    "\"fill\"",
+    "\"background\"",
+    "\"full-screen\"",
+    "\"focus\"",
+    "\"focuspoint\""
+  ],
+  "author": "\"Jonathon Menz <jonathonmenz.com> (http://jonathonmenz.com)\"",
```

Comment:
I think the `<jonathonmenz.com>` part should be removed because this is where the email would be.